score.cwl

```diff
@@ -4,92 +4,48 @@
 #
 cwlVersion: v1.0
 class: CommandLineTool
-baseCommand: python
+baseCommand: [Rscript, /score.R]
 
 hints:
   DockerRequirement:
-    dockerPull: python:3.8.8-slim-buster
+    dockerPull: docker.synapse.org/syn26720921/scoring:v1
 
 inputs:
-  - id: input_file
-    type: File
   - id: goldstandard
-    type: File
+    type: File?
```

Comment:
Are these inputs optional - the ones with `?`?  What happens if nothing gets passed to `score.R`?  I don't recall seeing any error catching in the script.

Reply:
yes, I have changed all inputs to "required". Thanks!